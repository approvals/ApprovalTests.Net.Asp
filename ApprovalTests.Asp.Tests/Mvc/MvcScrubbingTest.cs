using ApprovalTests.Asp.Mvc;
using ApprovalTests.Reporters;
using Microsoft.VisualStudio.TestTools.UnitTesting;

namespace ApprovalTests.Asp.Tests.Mvc
{
    [TestClass]
    [UseReporter(typeof (DiffReporter), typeof (FileLauncherReporter))]
    public class MvcScrubbingTest
    {
     

        [TestMethod]
        public void TestMvcPage()
        {
            MvcApprovals.VerifyMvcPage<TestableExampleController>(c => c.TestToken);
        }


    }

 
}