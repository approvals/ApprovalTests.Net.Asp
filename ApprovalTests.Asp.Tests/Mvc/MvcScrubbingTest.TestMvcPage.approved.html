

<!DOCTYPE html>

<html>
<head><base href="http://localhost:11625/TestableExample">
    <title>Verification Token Page</title>
</head>
<body>
<div>
    <!-- request verification token -->
</div>
</body>
</html>